src/tasks/minions/alchingActivity.ts

```diff
@@ -0,0 +1,69 @@
+import { AlchingActivityTaskOptions } from '../../lib/types/minions';
+import { KlasaMessage, Task } from 'klasa';
+import { channelIsSendable } from '../../lib/util/channelIsSendable';
+import { resolveNameBank, toKMB } from 'oldschooljs/dist/util';
+import { noOp, roll, saidYes } from '../../lib/util';
+import getUsersPerkTier from '../../lib/util/getUsersPerkTier';
+import { Time } from '../../lib/constants';
+import hasItemEquipped from '../../lib/gear/functions/hasItemEquipped';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import getOSItem from '../../lib/util/getOSItem';
+
+const { id: bryophytasStaffId } = getOSItem("Bryophyta's staff");
```

Comment:
`const bryophytasStaffId  = itemID("Bryophyta's staff")`
the itemID fn is made for this